frontend/src/features/admin-form/settings/components/EndPageSettingsSection/EndPageSettingsInput.tsx

```diff
@@ -0,0 +1,116 @@
+import { useCallback, useMemo } from 'react'
+import { useForm } from 'react-hook-form'
+import { FormControl, Stack } from '@chakra-ui/react'
+import validator from 'validator'
+
+import FormErrorMessage from '~components/FormControl/FormErrorMessage'
+import FormLabel from '~components/FormControl/FormLabel'
+import Input from '~components/Input'
+import Textarea from '~components/Textarea'
+
+import { useMutateFormSettings } from '../../mutations'
+
+interface EndPageSettingsInputProps {
+  settings: {
```

Comment:
why nest the info under a settings key?

Reply:
Brain fart... I updated the PR directly include the info instead of nesting it. Thanks!